.clang-tidy

```diff
@@ -0,0 +1,10 @@
+---
+Checks:          '-*,misc-throw-by-value-catch-by-reference,misc-static-assert'
```

Comment:
Is the idea that we append this list as we get stricter and stricter?

Reply:
exactly; every check I do (and add as a commit here) would be added to the list.